falcon/testing/client.py

```diff
@@ -750,6 +757,7 @@ async def _simulate_request_asgi(
     content_type: Optional[str] = None,
     body: Optional[Union[str, bytes]] = None,
     json: Optional[Any] = None,
+    msgpack: Optional[Any] = None,
```

Comment:
We have not marked these arguments as keyword-only, so theoretically this addition may be seen as a breaking change. Please move _msgpack_ to the end of the argument list.